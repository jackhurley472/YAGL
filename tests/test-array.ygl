int bar;
bar = 9;

int[10] foo;
foo[0] = 0;
foo[2+3] = 123;
foo[bar] = 456;

printInt(foo[3-3]);
printInt(foo[5]);
printInt(foo[bar]);

int temp;
temp = foo[0];
printInt(temp);
