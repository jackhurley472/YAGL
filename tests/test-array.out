0
123
456
0
